consensus/common/src/main/java/org/hyperledger/besu/consensus/common/bft/payload/PayloadHelpers.java

```diff
@@ -25,4 +26,14 @@ public static Hash hashForSignature(final Payload unsignedMessageData) {
         Bytes.concatenate(
             Bytes.of(unsignedMessageData.getMessageType()), unsignedMessageData.encoded()));
   }
+
+  public static Hash qbftHashForSignature(final Payload unsignedMessageData) {
```

Comment:
Would prefer that qbft specifc hashing wasn't in the common bft code and instead part of the qbft code

Reply:
agreed.